omnibus/config/software/datadog-agent-integrations.rb

```diff
@@ -88,35 +128,27 @@
       # We don't have auto_conf on windows yet
       if os != 'windows'
         if File.exist? "#{check_dir}/auto_conf.yaml"
-          mkdir check_conf_dir
-          copy "#{check_dir}/auto_conf.yaml", "#{check_conf_dir}/" unless File.exist? "#{check_conf_dir}/auto_conf.yaml"
+          mkdir check_conf_dir unless File.exists? (check_conf_dir)
+          copy "#{check_dir}/auto_conf.yaml", "#{check_conf_dir}/"
```

Comment:
why change these 2 lines? I think we should keep the condition `unless File.exist? "#{check_conf_dir}/auto_conf.yaml"` to make sure an `auto_conf` file override in the `datadog-agent` repo takes precedence over an `integrations-core` `auto_conf` file

Reply:
This is a bad rebase.